custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -2476,13 +2476,13 @@ def test_growth_tracker_export(self):
                       'N/A',
                       0,
                       0,
+                      'N/A',
```

Comment:
hey, do you know why did the test case change here when this is just a performance improvement change?

Reply:
It's a bit expected test change, earlier it used to show `DATA_NOT_ENTERED` even if the data doesn't exists for the show, which is not the right behaviour, it has to be "N/A".

Reply:
Is this change requested by someone? Or at least cleared with Nitish? It seems like a reasonable change, but people are very picky about what that text says and where, so it is good to talk to him before making these changes.